langchain_tools_demo/int.tests.cloudbuild.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - id: "Deploy to Cloud Run"
+    name: "gcr.io/cloud-builders/gcloud:latest"
+    dir: langchain_tools_demo
```

Comment:
Do we need to deploy the extension service as well?

Reply:
Instead of writing unit tests for the demo, the MVtesting is the build and deploy seen here. The extension service is not needed to test the demo deployment. The demo can handle errors when trying to contact the extension. We could update this in the future to be the full E2E test for both services